examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -0,0 +1,263 @@
+
+def index_to_xy_coord(x, y, z=10):
+    '''Transforms data index to coordinates of the background + offset'''
+    coords = coord_transform(x, y, z,
+                             affine=thresholded_score_map.get_affine())
+    return np.array(coords)[np.newaxis, :] + np.array([0, 1, 0])
+
+
+xy_indices_of_special_voxels = [(30, 10), (32, 10), (31, 9), (31, 10)]
+
+display = plot_stat_map(thresholded_score_map, bg_img=dataset.background,
+                        cut_coords=[-8], display_mode='z', aspect=1.25,
+                        title='Explained variance per voxel')
+
+# creating a marker for each voxel and adding it to the statistical map
+
+for i, (x, y) in enumerate(xy_indices_of_special_voxels):
+    display.add_markers(index_to_xy_coord(x, y),
+                        edgecolor=['r', 'b', 'magenta', 'g'][i],
+                        marker_size=150, marker = 's',
+                        facecolor = 'none', lw = 4.5)
+
+
+# re-set figure size after construction so colorbar gets rescaled too
+fig = plt.gcf()
+fig.set_size_inches(12, 12)
+
+plt.show()
+
+##############################################################################
+# Estimating receptive fields
+# ---------------------------
+# Now we take a closer look at the receptive fields of the four marked voxels.
+# A voxel's `receptive field <http://en.wikipedia.org/wiki/Receptive_field>`_
+# is the region of a stimulus (like an image) where the presence of an object,
+# like a white instead of a black pixel, results in a change in activity
+# in the voxel. In our case the receptive field is just the vector of 100
+# regression  coefficients (one for each pixel) reshaped into the 10x10
+# form of the original images. Some voxels are receptive to only very few
+# pixels, so we use `Lasso regression
+# <http://en.wikipedia.org/wiki/Lasso_(statistics)>`_ to estimate a sparse
+# set of regression coefficients.
+
+from sklearn.linear_model import LassoLarsCV
+
+# automatically estimate the sparsity by cross-validation
+lasso = LassoLarsCV(max_iter=10)
+
+# Mark the same pixel in each receptive field
+marked_pixel = (4, 2)
+
+from matplotlib import gridspec
+from matplotlib.patches import Rectangle
+
+fig = plt.figure(figsize=(12, 12))
+gs1 = gridspec.GridSpec(2, 3)
+
+# we fit the Lasso for each of the three voxels of the upper row
+for i, index in enumerate([1780, 1951, 2131]):
+    ax = plt.subplot(gs1[0, i])
+    # we reshape the coefficients into the form of the original images
+    rf = lasso.fit(y_train, X_train[:, index]).coef_.reshape((10, 10))
+    # add a black background
+    ax.imshow(np.zeros_like(rf), vmin=0., vmax=1., cmap='gray')
+    ax_im = ax.imshow(np.ma.masked_less(rf, 0.1), interpolation="nearest",
+                      cmap=['Reds', 'Greens', 'Blues'][i], vmin=0., vmax=0.75)
+    # add the marked pixel
+    ax.add_patch(Rectangle(
+        (marked_pixel[1] - .5, marked_pixel[0] - .5), 1, 1,
+        facecolor = 'none', edgecolor = 'r', lw = 4))
```

Comment:
PEP8 says that, in function calls, there should be no spaces before and after equal signs used for keyword-arguments (unlike equal signs used in assignments, outside the function calls.